app/src/auth0-config.ts

```diff
@@ -0,0 +1,6 @@
+const config = {
```

Comment:
Let's consider a config file in the root of the app folder.

Reply:
How do you want the config file to look like? Just `.config`?